src/zenml/constants.py

```diff
@@ -154,6 +155,7 @@ def handle_int_env_var(var: str, default: int = 0) -> int:
 ENV_ZENML_BACKUP_SECRETS_STORE_PREFIX = "ZENML_BACKUP_SECRETS_STORE_"
 ENV_ZENML_SKIP_PIPELINE_REGISTRATION = "ZENML_SKIP_PIPELINE_REGISTRATION"
 ENV_AUTO_OPEN_DASHBOARD = "AUTO_OPEN_DASHBOARD"
+ENV_ZENML_AUTO_OPEN_DASHBOARD = "ZENML_AUTO_OPEN_DASHBOARD"
```

Comment:
I guess this is a new version of the (now deprecated) "AUTO_OPEN_DASHBOARD" variable. I just realized that we didn't even have the old one in the docs, so it might be nice to add that to the environment variables page as well.

Reply:
I would keep the docs limited to the important env variables tbh. This is a niche env var, and everytime it actually affects you (when you login), the log message already tells you about it.